app/app.js

```diff
@@ -1,18 +1,21 @@
 const express = require("express");
-const { getAllTopics, apiDescription } = require('./app.controllers')
-const { handleNotFoundError, handleServerErrors } = require('./errors')
+const {
+  getAllTopics,
+  apiDescription,
+  getArticle,
+} = require("./app.controllers");
+const { handleNotFoundError, handleServerErrors } = require("./errors");
 
 const app = express();
 
-
-
 app.get("/api", apiDescription);
-app.get('/api/topics', getAllTopics);
+app.get("/api/topics", getAllTopics);
+app.get("/api/articles/:article_id", getArticle);
+app.get("/api/articles", getArticle);
```

Comment:
There should be separate controllers for task 4 and 5 rather than just using `getArticle`. You can keep `getArticle` for whichever one you want, but please create another controller that handles the other one. 👍  I would personally create another one called `getAllArticles` that handles task 5.